src/presentation/http/router/noteSettings.test.ts

```diff
@@ -10,68 +12,148 @@ describe('NoteSettings API', () => {
     await global.db.truncateTables();
   });
   describe('GET /note-settings/:notePublicId ', () => {
-    test('Returns note settings by public id with 200 status', async () => {
-      const user = await global.db.insertUser();
-
-      const note = await global.db.insertNote({
-        creatorId: user.id,
-      });
-
-      const noteSettings = await global.db.insertNoteSetting({
-        noteId: note.id,
+    test.each([
+      /** Returns 200 when the note public */
```

Comment:
we should not return note setting for all people. Only for note team with Write role